src/openrct2/scripting/bindings/ride/ScRide.cpp

```diff
@@ -584,10 +584,10 @@ namespace OpenRCT2::Scripting
         return ride != nullptr ? ride->drops & 0b11000000 : 0;
     }
 
-    uint8_t ScRide::highestDropHeight_get() const
+    float_t ScRide::highestDropHeight_get() const
     {
         auto ride = GetRide();
-        return ride != nullptr ? (ride->highest_drop_height >> 1) + (ride->highest_drop_height >> 2) : 0;
+        return ride != nullptr ? (ride->highest_drop_height * 0.75f) : 0;
```

Comment:
Maybe have this one as integers without any calculations, as it's the height in units which is also what the other height properties in the API return.

Reply:
Sure! I'm thinking of adding another case to formatString() to turn height units into their respective imperial / metric equivalents so plugin modders don't need to do that manually. Anything else that I'm missing?